Basics/Calexp_guided_tour.ipynb

```diff
@@ -0,0 +1,754 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `clone` method makes a deep copy. The result can be sliced with a BoundingBox"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "clone_cutout = calexp.clone()[bbox]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display1 = afw_display.Display(frame=1, backend='matplotlib')\n",
+    "display1.scale(\"asinh\", \"zscale\")\n",
+    "display1.mtv(clone_cutout.image)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Repeat for a coadd"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this last section, we will use Hyper Suprime-Cam (HSC) data that has been modified for tutorial purposes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coadd_butler = Butler('/project/shared/data/with-globular/')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coadd_butler.getKeys('deepCoadd_calexp')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We cannot use queryMetadata to look up what is available for coadds. This will be fixed in Butler Gen3."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the time being, open a terminal and list files in `/project/shared/data/with-globular/` to see what's available. Or just carry on using the following example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataId = {'filter':'HSC-I', 'tract':9813, 'patch':'4,4'}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Retrieve a coadd `calexp`, and see what methods it provides."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coadd = coadd_butler.get('deepCoadd_calexp', dataId)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coadd_methods = [m for m in dir(coadd) if not m.startswith('_')]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "coadd_methods"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "set(coadd_methods).symmetric_difference(set(calexp_methods))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result of the `set` command above shows that a calexp and a coadd have the same methods."
```

Comment:
That's because they're the same class, `lsst.afw.image.Exposure`.

Reply:
Oh, I see now, that makes this result expected!

Coadds have the same methods, but it seems to me that they have different contents...e.g. XY0 is non-zero; many more cosmic rays in the masks; overall masks look much different; PSFs must be an average from the coaddition process? And certainly the dataIds are a lot different. Maybe this part of the notebook needs to focus on those aspects.